latch_cli/main.py

```diff
@@ -66,6 +67,9 @@ def register(pkg_root: str, disable_auto_version: bool, remote: bool):
     except Exception as e:
         CrashReporter.report(pkg_path=pkg_root)
         click.secho(f"Unable to register workflow: {str(e)}", fg="red")
+    
+    # remove cached crash-report logs
+    shutil.rmtree(pkg_root + 'logs/')
```

Comment:
move this line into the body of the `latch_cli.services.register` function if possible - if not, put it in a `finally` clause

Reply:
I think we need to keep it in `main.py` bc we need to remove it after `CrashReporter()` in case it is generated.

Also, I tried using `finally`, but I get an error because `finally` is executed before `try: register()` finishes executing (aka before we even generate the logs) since `register()` takes a while. 

Ideally, we want it to execute after `register()` has returned, so I'm not sure `finally` can work.

Any thoughts?

Reply:
Is the issue you're running into that the crash reporter is trying to include log files that might not be there? What I'm picturing in my head is a user flow like below:

Run latch register -> something breaks before the logs are generated -> CrashReporter fails since it tries to bring in log files that aren't there.

If this is what you have in mind, I think an easy fix is to make CrashReporter try to include log files, and just skip them if they don't exist. If you do this then it shouldn't break if you move the `shutil.rmtree(...)` line. Let me know if this isn't what you were describing though.

Reply:
Okay, I added a check in `finally:` in case logs aren't there. It seems to work consistently.